plugin-flex-ts-template-v2/src/feature-library/supervisor-barge-coach/custom-components/SupervisorAlertButton/SupervisorAlertButtonComponent.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect } from "react";
+import { ITask, IconButton } from '@twilio/flex-ui';
+import { useDispatch, useSelector } from 'react-redux';
+import { reduxNamespace } from '../../../../utils/state';
+import { AppState } from '../../../../types/manager';
+import { Actions } from "../../flex-hooks/states/SupervisorBargeCoach"
+import { Flex, Tooltip } from "@twilio-paste/core";
+import { updateTaskAndTriggerAlerts, syncUpdates } from "../../helpers/supervisorAlertHelper"
+
+type SupervisorAlertButton = {
+  task: ITask
+}
+
+export const SupervisorAlertButton = ({task}: SupervisorAlertButton) => {
+  const dispatch = useDispatch();
+
+  const {
+    enableAgentAssistanceAlerts,
+    agentAssistanceSyncSubscribed
+  } = useSelector((state: AppState) => state[reduxNamespace].supervisorBargeCoach);
+
+  const agentAssistanceAlertToggle = () => {
+    if (enableAgentAssistanceAlerts) {
+      dispatch(Actions.setBargeCoachStatus({ 
+        enableAgentAssistanceAlerts: false
+      }));
+      // If the supervisor disabled the agent assistance alerts, let's cache this
+      // to ensure it is set to false if a browser refresh happens
+      localStorage.setItem('cacheAlerts',"false");
+    } else {
+      dispatch(Actions.setBargeCoachStatus({ 
+        enableAgentAssistanceAlerts: true
+      }));
+      updateTaskAndTriggerAlerts();
+      localStorage.setItem('cacheAlerts',"true");
+    }
+  }
+
+  useEffect(() => {
+    if(!agentAssistanceSyncSubscribed){
+      syncUpdates();
+    }
+  });
+  // Return the Supervisor Agent Assistance Toggle, this gives the supervisor
+  // the option to enable or disable Agent Assistance Alerts
+  return (
+   <Tooltip text={enableAgentAssistanceAlerts ? "Agent Assistance Alerts Enabled" : "Agent Assistance Alerts Disabled"} placement="left">
+      <Flex vAlignContent="center">
+          <IconButton
+            icon={ enableAgentAssistanceAlerts ? 'BellBold' : 'Bell' }
+            onClick={() => agentAssistanceAlertToggle()}
+            style={{width:'20px',height:'20px'}}
```

Comment:
You can replace the sizings here with `size="small"`. However, the colors are incorrect when compared to the other IconButtons in the MainHeader. It looks like you will want to apply this style: https://code.hq.twilio.com/flex/flex-monorepo/blob/99f28dff50df44f3d33a5d4195a548ab6f674fa1/packages/flex-ui/src/components/MainHeader/MainHeader.Components.ts#L47-L56